easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-parallelbuild.patch

```diff
@@ -0,0 +1,80 @@
+diff -ru nwchem-6.6.orig/src/config/makefile.h nwchem-6.6/src/config/makefile.h
```

Comment:
Done. It was taken from the existing NWchem 6.5 of course but re-done with diff to match correctly and i forgot to re-add the comment at the top :-(